reactjs-films-homework/src/pages/ActorProfile/ActorTitleInfo/ActorTitleInfo.tsx

```diff
@@ -19,6 +21,9 @@ export interface IActorTitleProps {
 const ActorTitle: React.FC<IActorTitle> = ({ props }) => {
   const { birthday, name, profile_path, biography, place_of_birth } = props;
   const urlImg = `https://image.tmdb.org/t/p/w500/${profile_path}`;
```

Comment:
such urls should be stored in some common constants and imported to the files.
As I see, this URL is repeated in several files.